README.md

```diff
@@ -31,3 +30,102 @@ After that create a `.eslintrc.json` file and add the following:
 We encourage everybody to start discussions and file PRs with changes to this rule set.
 However please always make a good point, why you want to introduce a change.
 Simply "not liking" something doesn't count.
+
+# Changelog
```

Comment:
I'd prefer to have the changelog in a separate file. Blows up the readme otherwise